pkgs/by-name/pr/proftpd/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  glibc,
+  libcap,
+  libsodium,
+  openssl,
+  zlib,
+  libxcrypt-legacy,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "proftpd";
+  version = "1.3.8c";
+
+  src = fetchFromGitHub {
+    owner = "proftpd";
+    repo = "proftpd";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-wHcJJiGY8ULVIMo6RVEE7ne6YZM7gECt2IygPhX+3cU=";
+  };
+
+  buildInputs = [
+    libcap
+    libsodium
+    openssl
+    zlib
+    libxcrypt-legacy
+  ];
+
+  patches = [ ./proftpd.patch ];
+
+  preConfigure = ''
+    #sed -i s#/usr/include/linux#${glibc}/include/linux# lib/libcap/Makefile
```

Comment:
I assume this is dead code and can be removed?
Otherwise, please use `substituteInPlace` with `--replace-fail`, then we'd notice on an upgrade that the replacement needs an update.